### DISCUSSION TITLE
problem with citations

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#4014] [Q&A]
take e.g. this case from the latest 'citations' file: 
cluster_id 13914752	:    488 U.S. 984	
the citation shows Levitt v Univ of Texas according to Google Scholar https://scholar.google.com/scholar?hl=en&as_sdt=800006&q=488%09U.S.%09984&btnG=
It should be Pollard v Owens according to CL web search https://www.courtlistener.com/opinion/9083109/pollard-v-owens-illinois-inc/
and it is Armentero v. Makel  if you search for this cluster_id in the bulk files (opinions+cluster+docket)  
i have found several of these mismatches  
is 'cluster_id' the same in both opinion_cluster and citations files?
what is the most reliable source for citations?  and if it is not by cluster_id , any recommendation how else to merge citations with opinions? 
thank you!

**Comment by mlissner** [comment]
I think the answer you're looking for is that the bulk citations file contains links between opinions. Does that fix these problems?

**Comment by legaltextai** [comment]
Not sure I understand.
When I join by cluster_id I run into the issue I described.
If that’s not the way to merge the files, how do I connect the name of the
case to its proper citation?

**Comment by mlissner** [comment]
My first response misunderstood your question, so I hid it. 

---

488 U.S. 984 can be several cases: 

https://cite.case.law/us/488/984/

This is because each of the cases is just one or two lines long and the publisher put several on the same page of the book. 

When I go to the cluster ID you provided: 

https://www.courtlistener.com/opinion/13914752/x/

It throws a 404, so that's weird.

>  is 'cluster_id' the same in both opinion_cluster and citations files?

Yes.

So ultimately, if you want to merge a citation with a cluster, you'll want to join the cluster_id in the citation object with the id field of the cluster object.

**Reply by legaltextai** [reply]
yeah, I did not realize several cases may be on the same page in the reporter. I excluded all opinions < 200 words, ran a random check on citations, looks fine now.
